pkg/tarmak/provider/amazon/amazon.go

```diff
@@ -323,6 +325,7 @@ func (a *Amazon) Environment() ([]string, error) {
 		fmt.Sprintf("AWS_SECRET_ACCESS_KEY=%s", creds.SecretAccessKey),
 		fmt.Sprintf("AWS_SESSION_TOKEN=%s", creds.SessionToken),
 		fmt.Sprintf("AWS_DEFAULT_REGION=%s", a.Region()),
+		fmt.Sprintf("EBS_VOLUME_ENCRYPTED=%s", fmt.Sprint(*a.tarmak.Cluster().Config().Amazon.EBSEncrypted)),
```

Comment:
This needs to use the method rather than the struct directly (`AmazonEBSEncrypted()`)